### PR TITLE
slight rework

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**

### CODE REVIEW

File: src/Components/AvailableBidder/AvailableBidderTable/AvailableBidderTable.jsx [hunk 1]
```diff
@@ -111,15 +110,10 @@ const AvailableBidderTable = props => {
...
-  const internalCDAExportToggle = () => {
-    setInternalViewToggle(!internalViewToggle);
-    setInternalCDAView(!internalCDAView);
-  };
-
   const exportBidders = () => {
     if (!isLoading) {
       setExportIsLoading(true);
-      availableBidderExport(internalCDAView, sort)
+      availableBidderExport(isInternalCDA && internalViewToggle, sort)
```
Comment (SidelineCory24) [inline src/Components/AvailableBidder/AvailableBidderTable/AvailableBidderTable.jsx:116]: 😎

File: src/Components/AvailableBidder/AvailableBidderTable/AvailableBidderTable.jsx [hunk 2]
```diff
@@ -206,7 +200,7 @@ const AvailableBidderTable = props => {
                             </Tooltip>
                           }
                           checked={!internalViewToggle}
-                          onChange={() => internalCDAExportToggle()}
+                          onChange={() => setInternalViewToggle(!internalViewToggle)}
```
Comment (SidelineCory24) [inline src/Components/AvailableBidder/AvailableBidderTable/AvailableBidderTable.jsx:203]: 🐐

**Review by SidelineCory24 — APPROVED** [review]
Looks good to go!🚀

**Comment by elizabeth-jimenez** [comment]
> Looks good to go!🚀

Now it does 😁  👨‍💻  thanks Cory!

**Pull request merged**